.gitignore

```diff
@@ -45,7 +45,7 @@
 # Ignore application configuration
 /config/application.yml
 /config/database.yml
-.env.local
+.env*
```

Comment:
I don't think the sample environment file should be ignored.  We should probably be very explicit here, rather than "glob"ularly inclusive.  It is a sample, after all, that is why you wanted the sample, right?

Reply:
@kotp its not being ingored - The sample file is named `env.sample` without a `.` at the beginning. Instructions say to copy `env.sample` to `.env.whatever`

Reply:
Oh, I missed that `env.sample` did not have the `dot`.